src/Sentry/SentryLogLevel.cs

```diff
@@ -0,0 +1,75 @@
+using Sentry.Extensibility;
+using Sentry.Infrastructure;
+
+namespace Sentry;
+
+/// <summary>
+/// The severity of the structured log.
+/// <para>This API is experimental and it may change in the future.</para>
+/// </summary>
+/// <seealso href="https://develop.sentry.dev/sdk/telemetry/logs/"/>
```

Comment:
Maybe worth adding something to the comments here about the significance of the `int` values assigned to each of the enum members. I was initially a bit surprised we were assigning values here... only worked out why by reading these docs:
- https://develop.sentry.dev/sdk/telemetry/logs/#log-severity-level